sections/semantics-tabular-data.include

```diff
@@ -1075,6 +1075,52 @@
 
   The <{tfoot}> element takes part in the <a>table model</a>.
 
+  <div class="example">
+    The following example uses a <{tfoot}> to indicate a summary of the total monthly spending,
+    earnings, and holdings.
+
+    <xmp highlight="html">
+      <table>
+        <thead>
+          <tr>
+            <th scope="col">Month</th>
+            <th scope="col">Spending</th>
+            <th scope="col">Earnings</th>
+            <th scope="col">Holdings</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th scope="row">January</th>
+            <td>$800</td>
+            <td>$700</td>
+            <td>-$100</td>
+          </tr>
+          <tr>
+            <th scope="row">February</th>
+            <td>$900</td>
+            <td>$935</td>
+            <td>+$35</td>
+          </tr>
+          <tr>
+            <th scope="row">March</th>
+            <td>$900</td>
+            <td>$950</td>
+            <td>+$50</td>
+          </tr>
+        </tbody>
+        <tfoot>
```

Comment:
Do you want to `<mark>` the tfoot here? It's a big example...

(that means switching to use pre... so you may not.

Reply:
mmm...it is big... maybe i cut out some of the `tr`s instead...and people will just have to assume my math is correct?

otherwise, yeh.  Probably makes sense to use `mark` and `pre`.  If you don't like the above mentioned idea, @chaals, i'll move to `pre`.

Reply:
Either way. I have a minor preference for `mark`ing things in examples. Pik one, do it, and feel free to merge when it builds clean.